packages/core/src/lib/androidSdk/AndroidSdkTools.ts

```diff
@@ -148,8 +148,8 @@ export class AndroidSdkTools {
     const apkSignerParams = [
       `sign --ks ${keystore}`,
       `--ks-key-alias ${alias}`,
-      `--ks-pass pass:${ksPass}`,
-      `--key-pass pass:${keyPass}`,
+      `--ks-pass "pass:${ksPass}"`,
```

Comment:
Shouldn't we be using single quotes instead of double quotes? Double quotes means that the password can contain spaces, but single quotes will escape other characters like `*` and `.`.